images/linuxptp-daemon.yml

```diff
@@ -24,6 +24,7 @@ enabled_repos:
 for_payload: false
 from:
   builder:
+  - member: openshift-enterprise-base
   - stream: golang
```

Comment:
This update triggered an incorrect change in https://github.com/openshift/linuxptp-daemon/pull/155/files
The new line for builder was inserted in wrong order.  The correct order should be:
```
  builder:
  - stream: golang
  - member: openshift-enterprise-base
```